controller/coq_lsp.ml

```diff
@@ -318,7 +318,6 @@ let dispatch_request ~method_ ~params : RAction.t =
   (* Lifecyle *)
   | "initialize" ->
     LIO.trace "dispatch_request" "duplicate initialize request! Rejecting";
-    (* XXX what's the error code here *)
```

Comment:
Actually that's a real TODO, `-32600` is likely not the right code.

Why do we mix unrelated changes in PRs?